CHANGELOG.md

```diff
@@ -37,6 +37,10 @@ Additional labels for pre-release and build metadata are available as extensions
    1. `PTH` (Issue [#247](https://github.com/IEA-Task-43/digital_wra_data_standard/issues/247))
 1. To `measurement_location` add:
    1. `uuid` (Issue [#232](https://github.com/IEA-Task-43/digital_wra_data_standard/issues/232))
+1. To `calibration` add:
+   1. `slope_unit` (Issue [#248](https://github.com/IEA-Task-43/digital_wra_data_standard/issues/248))
+   1. `offset_unit` (Issue [#248](https://github.com/IEA-Task-43/digital_wra_data_standard/issues/248))
+   1. `sensitivity_unit` (Issue [#248](https://github.com/IEA-Task-43/digital_wra_data_standard/issues/248))
```

Comment:
You have put these changes into the previous release. They need to go into the "Unreleased" changelog above.